src/runtime/contrib/tensorrt/tensorrt_ops.cc

```diff
@@ -723,6 +723,53 @@ class ConcatOpConverter : public TensorRTOpConverter {
   }
 };
 
+class SplitOpConverter : public TensorRTOpConverter {
```

Comment:
We should add `#if TRT_VERSION_GE(5, 1, 5)` before it because the code uses `params->network->addSlice`.